src/template/partials/footer.html

```diff
@@ -1,3 +1,52 @@
 <footer>
+  <!-- Start Footer -->
+  <div class="footer">
+  
+    <div class="container">
+        <div class="download">
+            <h3>Grow your business fast <br>with Sark SaaS Landing.</h3>  
+                <button name="Download" value="Download">Download Now</button>
+        </div>
+        <div class="links">
+            <ul >
+                <li class="active"> Company </li>
+                <li><a href="#" >About</a> </li>
+                <li><a href="#" >Works          </a> </li>
+                <li><a href="#" >Features          </a> </li>
+                <li><a href="#" >Career            </a> </li>
+            </ul>    <ul>
+                <li class="active"> Help</li>
+                <li><a href="#" >Customer Support</a> </li>
+                <li><a href="#" >Delivery Details</a> </li>
+                <li><a href="#" >Terms & Conditions</a> </li>
```

Comment:
![zzz](https://github.com/shaban011147/landing/assets/106445705/4a3a1a36-de23-461c-9881-a249944e71e8)
not responsive

Reply:
done